internal/resources/azure/data_factory_integration_runtime_managed.go

```diff
@@ -0,0 +1,103 @@
+package azure
+
+import (
+	"fmt"
+
+	"github.com/infracost/infracost/internal/resources"
+	"github.com/infracost/infracost/internal/schema"
+	"github.com/shopspring/decimal"
+)
+
+// DataFactoryIntegrationRuntimeManaged struct represents Data Factory's Managed
+// VNET integration runtime.
+//
+// Resource information: https://azure.microsoft.com/en-us/services/data-factory/
+// Pricing information: https://azure.microsoft.com/en-us/pricing/details/data-factory/data-pipeline/
+type DataFactoryIntegrationRuntimeManaged struct {
```

Comment:
Do we need this as a shared resource or could we have `azurerm_data_factory_integration_runtime_managed ` build a `DataFactoryIntegrationRuntimeAzureSSIS`?

Reply:
It is marked as deprecated in v3 provider. I decided to duplicate the logic as manage runtime will probably go away soon and it will be easier to clean up.

Reply:
👍 might be good to note that in the type comment.

Reply:
Will do, thank you! 👍